metadata-ingestion/docs/sources/dataplex/dataplex_pre.md

```diff
@@ -0,0 +1,190 @@
+| :--------------- | :----------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------- |
+| Project          | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | GCP Project containing Dataplex resources                                                             |
+| Lake             | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Data lake container (sub-container of Project)                                                        |
+| Zone             | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Zone container under a Lake (RAW or CURATED type)                                                     |
+| Asset            | [`Container`](https://docs.datahub.com/docs/generated/metamodel/entities/container/) | Asset container representing a BigQuery dataset or GCS bucket (sub-container of Zone)                 |
+| Entity           | [`Dataset`](https://docs.datahub.com/docs/generated/metamodel/entities/dataset)      | Discovered table or fileset (linked to Asset container). Schema metadata is extracted when available. |
+
+#### Hierarchy
+
+The connector creates a hierarchical container structure:
+
+```
+Project (Container)
+└── Lake (Container)
+    └── Zone (Container)
+        └── Asset (Container)
+            └── Entity (Dataset)
+```
+
+#### Sibling Relationships
+
+When `create_sibling_relationships` is enabled, the connector creates bidirectional sibling relationships between Dataplex entities and their source platform entities (e.g., BigQuery tables, GCS objects). This allows you to:
+
+- Navigate between Dataplex and source platform views of the same data
+- Control which entity is considered primary using `dataplex_is_primary_sibling` config
+- By default, the source platform entity (BigQuery/GCS) is marked as primary
+
+#### Lineage
+
+When `extract_lineage` is enabled and proper permissions are granted, the connector extracts lineage information using the Dataplex Lineage API to capture relationships between:
+
+- Entities and their transformations
+- Data flows between assets
+- Processing workflows
+
+**Lineage Configuration Options:**
+
+- **`extract_lineage`** (default: `true`): Enable table-level lineage extraction
+- **`lineage_fail_fast`** (default: `false`): If true, fail immediately on lineage extraction errors. If false, log errors and continue processing. Recommended: `false` for development, `true` for production
+- **`max_lineage_failures`** (default: `10`): Maximum consecutive lineage extraction failures before stopping (circuit breaker). Set to `-1` to disable circuit breaker
```

Comment:
I don't recall this kind of parameters in other sources. In general, we warn and continue, that is `lineage_fail_fast: false`. In those cases where the API is flaky, we usually do retries but still warn and continue if failing after retries.

Is there any particular reason for having these parameters for Dataplex API? Have you found it to be very flaky or how is that?